### ISSUE TITLE
InCite needs footnotes/endnotes option and scrolling

### ISSUE THREAD

**Issue by Fozzle7907** [jimmejardine/qiqqa-open-source#275]
I love Qiqqa, great program and very intuitive, however, I need the option to cite in footnotes/endnotes. I know I can copy snippet and add it in, but it is laborious, especially with suggestions. Also, the scrolling of PDFs using two fingers in the mousepad is terrible, which is frustrating. When you use that form of scrolling on everything else, then invariably you use it with Qiqqa and it is just too sensitive and I end up spending time trying to find my reading place again. 

Thanks again for a wonderful program.

**Comment by GerHobbelt** [comment]
Note to self: sideways related: #280.

**Comment by nikilito** [comment]
I agree with this issue and I think it's one of the more important problems during a PDF reading stage!
The scrolling is not "fluid" as other program but rather jerkily and too sensitive.

Just to give additional feedback to the problem.
No hurry! 😄

Bye.
